developer/src/kmc-keyboard/test/test-keys.ts

```diff
@@ -10,92 +10,97 @@ const K = Constants.USVirtualKeyCodes;
 
 describe('keys', function () {
   this.slow(500); // 0.5 sec -- json schema validation takes a while
+  testCompilationCases(KeysCompiler, [
+    {
+      // should compile minimal keys data
+      subpath: 'sections/keys/minimal.xml',
+      callback(sect) {
+        const keys = <Keys> sect;
+        assert.ok(keys);
+        assert.equal(compilerTestCallbacks.messages.length, 0);
+        assert.equal(keys.keys.length, 1);
+        assert.equal(keys.flicks.length, 1); // there's always a 'null' flick
+        assert.equal(keys.keys[0].to.value, '🪦');
```

Comment:
Can we use a Unicode value here rather than a char, as I get tofu on Windows by default.

Reply:
done